features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -85,6 +85,11 @@
     </config>
   </feature>
 
+  <feature name="openhab.tp-jbbp" description="Java Binary Block Parser library" version="${project.version}">
+    <capability>openhab.tp;feature=jbbp;version=1.4.1</capability>
+    <bundle dependency="true">wrap:mvn:com.igormaznitsa/jbbp/1.4.1</bundle>
```

Comment:
@J-N-K Am I right that instead of wrapping the artifact here, it should be added to https://github.com/openhab/openhab-osgiify?

Reply:
@J-N-K Did you see my question?

Reply:
Already there. Just need to change to 

```
<bundle dependency="true">mvn:org.openhab.osgiify/com.igormaznitsa.jbbp/1.4.1</bundle>
```

Reply:
Excellent, thanks!

Reply:
```suggestion
    <bundle dependency="true">mvn:org.openhab.osgiify/com.igormaznitsa.jbbp/1.4.1</bundle>
```

Reply:
done